packages/openapi-to-graphql/src/resolver_builder.ts

```diff
@@ -418,6 +420,50 @@ export function getResolver({
             }
           }
 
+          // Apply autogenerated slicing arguments
+          if (data.options.addSlicingPattern && Array.isArray(saneData)) {
+            // NOTE: Does not differentiate between autogenerated args and
+            // preexisting args
+            if ('first' in args && 'last' in args) {
+              reject(
+                new Error(
+                  `Cannot use both 'first' and 'last' slicing arguments at the same time`
+                )
+              )
+            } else {
+              let arraySaneData = saneData as Array<any>
+
+              if ('first' in args) {
+                if (args['first'] >= 0) {
+                  arraySaneData = arraySaneData.slice(0, args['first'])
```

Comment:
Hm, I am wondering about this: By cutting the array returned from the REST API, I think we easily confuse users, because it is now unclear whether, for example, `last` relates to the last n entries of the first q entries fetched from the REST API, or to the last entries of all the data?

Reply:
As mentioned, I provided a link to a connections pattern specification provided by Facebook. 

If we deem it okay to adapt the specification to suit the slicing pattern, then what I have implemented should be fine because according to the specification, `first` and `last` affects the returned data, not the ends of the actual data. 

Something else to keep in mind is I’m hoping to use the slicing pattern as a stepping stone to implement the connections pattern.

Reply:
I am still not on board with the semantics of `first` and `last` you introduce here. They relate to (arbitrary) slices of data returned by the underlying REST call. In contrast, I think a user of a GraphQL native API would expect these to return the `first` or `last` entries of all data, or of a slice of data defined by a `cursor`.

Reply:
I.e., I would be much more in favor of implementing slicing using a `limit` argument, which does not imply _which_ part of a list of data is returned.